examples/workflows/scripts/callable_script.py

```diff
@@ -89,7 +91,7 @@ def function_kebab_object(annotated_input_value: Annotated[Input, Parameter(name
 with Workflow(name="my-workflow") as w:
     with Steps(name="my-steps") as s:
         my_function(arguments={"input": Input(a=2, b="bar", c=42)})
-        str_function(arguments={"input": Input(a=2, b="bar", c=42).json()})
+        str_function(arguments={"input": serialize(Input(a=2, b="bar", c=42))})
```

Comment:
This needs to be changed to `serialize` function in hera, which correctly handles dumping both v1 and v2 objects.